docs/Plugins/gitextractor.md

```diff
@@ -38,6 +38,7 @@ curl --location --request POST 'localhost:8080/pipelines' \
 ```
 - `url`: the location of the git repository. It should start with `http`/`https` for a remote git repository and with `/` for a local one.
 - `repoId`: column `id` of  `repos`.
+   Note : To find the repo id run `$("meta[name=octolytics-dimension-repository_id]").getAttribute('content')` in console. 
```

Comment:
To be more specific, this  command should be executed in the **browser console**

Reply:
Yes in the browser console